docs/src/components/icons-list.astro

```diff
@@ -37,4 +62,18 @@ const icons = Object.keys(Icons) as (keyof typeof Icons)[];
 		margin: 0;
 		padding: 0.75rem;
```

Comment:
Now the `<div>` has been changed for a `<button>`, we need to reset its default background properties.
```suggestion
		padding: 0.75rem;
		background: none;
```